decidim-core/app/uploaders/decidim/avatar_uploader.rb

```diff
@@ -7,11 +7,15 @@ class AvatarUploader < ImageUploader
 
     process :validate_dimensions
 
-    version :big do
+    version :profile do
+      process resize_to_fit: [268, 320]
```

Comment:
Not sure about this. Should either be `resize_to_fill` with the exact versions specified, or `resize_to_fit` with only the image's width (otherwise it will look weird in an image with a big height).

Reply:
But it will resize and loose quality. I believe we should go with `resize_to_fit: [268]` then.

Reply:
Done!